docs/content/guides/rows/rows-sorting.md

```diff
@@ -3171,28 +3141,24 @@ ReactDOM.render(<App />, document.getElementById('exampleExcludeRowsFromSorting'
 
 ## Control sorting programmatically
 
-You can control sorting at the grid's runtime by using Handsontable's
-[hooks](@/guides/getting-started/events-and-hooks.md) and
+You can control sorting at the grid's runtime by using Handsontable's [hooks](@/guides/getting-started/events-and-hooks.md) and
 [API methods](@/api/columnSorting.md#methods).
 
 This allows you to:
 
-- Enable or disable sorting depending on specified conditions. For example, you can disable sorting
-  for very large data sets.
-- Trigger sorting depending on the state of another component in your application. For example, you
-  can let the end user sort data by clicking on buttons outside of the grid.
+- Enable or disable sorting depending on specified conditions. For example, you can disable sorting for very large data sets.
```

Comment:
Hello, I would propose the following version to be considered:

"Enable or disable sorting functionality by using specified conditions, such as size, height, lenght etc."